le_dash/settings/base.py

```diff
@@ -150,3 +150,14 @@
 }
 
 BANNER_BASE_URL = getenv('BANNER_BASE_URL')
+
+# Use local redis server to cache searches or template fragments,
+# timeout is 1 hour
+# Use KEY_PREFIX for production
+CACHES = {
```

Comment:
I want to figure out the authentication before we cache the higher level stuff. I cache the queries first because multiple views share the same queries.